examples/vrf-anchor/README.md

```
@@ -0,0 +1,34 @@
+# vrf-anchor
+
+A minimal **Anchor** example of verifiable randomness (VRF):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)